Struggled with cmake not building the tests correctly. Kept getting the following error:
```CMake Error at /Users/astamile/opt/anaconda3/envs/isis/share/cmake-3.23/Modules/GoogleTestAddTests.cmake:112 (message):
CMake Error at /Users/astamile/opt/anaconda3/envs/isis/share/cmake-3.23/Modules/GoogleTestAddTests.cmake:112 (message):
  Error running test executable.

    Path: '/Users/astamile/isis/ISIS3/build/tests/runISISTests'
    Result: Segmentation fault
```
I was able to run the unit tests with the following failures:
```
	128 - isis_unit_test_JP2Importer (Failed)
	321 - isis_unit_test_SqlRecord (Failed)
	1043 - mex_unit_test_MexHrscSrcCamera (Failed)
```

They don't seem applicable to these PR changes.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -40,6 +40,8 @@ release.
 - Updated the LRO calibration application Lrowaccal to add a units label to the RadiometricType keyword of the Radiometry group in the output cube label if the RadiometricType parameter is Radiance. No functionality is changed if the RadiometricType parameter is IOF. Lrowaccal has also been refactored to be callable for testing purposes. Issue: [#4939](https://github.com/USGS-Astrogeology/ISIS3/issues/4939), PR: [#4940](https://github.com/USGS-Astrogeology/ISIS3/pull/4940)
 - Changed how logs are reported so they no longer only printing at the end of the applications execution. [#4914](https://github.com/USGS-Astrogeology/ISIS3/issues/4914)
 - Update marcical to include step 3 of the mission team's MARCI calibration process described [here](https://pds-imaging.jpl.nasa.gov/data/mro/mars_reconnaissance_orbiter/marci/mrom_1343/calib/marcical.txt). [#5004](https://github.com/USGS-Astrogeology/ISIS3/pull/5004)
+
+
 ### Added
 - Improved functionality of msi2isis and MsiCamera model to support new Eros dataset, including support for Gaskell's SUMSPICE files that adjust timing, pointing and spacecraft position ephemeris. [#4886](https://github.com/USGS-Astrogeology/ISIS3/issues/4886)
 - Added a new application, framestitch, for stitching even and odd push frame images back together prior to processing in other applications. [4924](https://github.com/USGS-Astrogeology/ISIS3/issues/4924)
@@ -55,6 +57,8 @@ release.
 - Fixed algorithm for applying rolling shutter jitter. Matches implementation in USGSCSM.
 - Fixed isis2std incorrectly outputing signed 16 bit tiff files. [#4897](https://github.com/USGS-Astrogeology/ISIS3/issues/4897)
 - Fixed CNetCombinePt logging functionality such that only merged points are included in the log. [#4973](https://github.com/USGS-Astrogeology/ISIS3/issues/4973)
+- Removed SpkCenterId functions in Cassini camera models due to spkwriter writing positions of Cassini relative to Titan but labeling
+it in the kernel as the position relative to the Saturn Barycenter. [#4942](https://github.com/USGS-Astrogeology/ISIS3/issues/4942)
 
 
 ## [7.0.0] - 2022-02-11

--- a/isis/src/cassini/objs/IssNACamera/IssNACamera.h
+++ b/isis/src/cassini/objs/IssNACamera/IssNACamera.h
@@ -59,6 +59,10 @@ namespace Isis {
    *   @history 2015-10-16 Ian Humphrey - Removed declarations of spacecraft and instrument
    *                           members and methods and removed implementation of these methods
    *                           since Camera now handles this. References #2335.
+   *   @history 2022-07-14 Amy Stamile - Removed SpkCenterId function due to spkwriter writing
+   *                           positions of Cassini relative to Titan (NAIF ID 606) but labeling
+   *                           it in the kernel as the position relative to the Saturn Barycenter
+   *                           (NAIF ID 6) Reference #4942.
    */
   class IssNACamera : public FramingCamera {
     public:
@@ -83,14 +87,6 @@ namespace Isis {
        *         Kernel Reference ID
        */
       virtual int CkReferenceId() const { return (1); }
-
-      /**
-       *  SPK Center ID - 6 (Saturn)
-       *
-       * @return @b int The appropriate instrument code for the Spacecraft
-       *         Kernel Center ID
-       */
-      virtual int SpkCenterId() const { return 6; }
 
       /**
        *  SPK Reference ID - J2000

--- a/isis/src/cassini/objs/IssWACamera/IssWACamera.h
+++ b/isis/src/cassini/objs/IssWACamera/IssWACamera.h
@@ -60,6 +60,10 @@ namespace Isis {
    *   @history 2015-10-16 Ian Humphrey - Removed declarations of spacecraft and instrument
    *                           members and methods and removed implementation of these methods
    *                           since Camera now handles this. References #2335.
+   *   @history 2022-07-14 Amy Stamile - Removed SpkCenterId function due to spkwriter writing
+   *                           positions of Cassini relative to Titan (NAIF ID 606) but labeling
+   *                           it in the kernel as the position relative to the Saturn Barycenter
+   *                           (NAIF ID 6) Reference #4942.
    */
   class IssWACamera : public FramingCamera {
     public:
@@ -84,14 +88,6 @@ namespace Isis {
        *         Kernel Reference ID
        */
       virtual int CkReferenceId() const { return (1); }
-
-      /**
-       *  SPK Center ID - 6 (Saturn)
-       *
-       * @return @b int The appropriate instrument code for the Spacecraft
-       *         Kernel Center ID
-       */
-      virtual int SpkCenterId() const { return 6; }
 
       /**
        *  SPK Reference ID - J2000

--- a/isis/src/cassini/objs/VimsCamera/VimsCamera.h
+++ b/isis/src/cassini/objs/VimsCamera/VimsCamera.h
@@ -72,6 +72,10 @@ namespace Isis {
    *   @history 2018-03-14 Adam Goins - Changed cache calculations with LoadCache() call.
    *                           This fixes an error where VimsCamera caused spiceinit to
    *                           fail when TargetName == SKY. Fixes #5353.
+   *   @history 2022-07-14 Amy Stamile - Removed SpkCenterId function due to spkwriter writing
+   *                           positions of Cassini relative to Titan (NAIF ID 606) but labeling
+   *                           it in the kernel as the position relative to the Saturn Barycenter
+   *                           (NAIF ID 6) Reference #4942.
    */
   class VimsCamera : public Camera {
     public:
@@ -108,14 +112,6 @@ namespace Isis {
        *         Kernel Reference ID
        */
       virtual int CkReferenceId() const { return (1); }
-
-      /**
-       *  SPK Center ID - 6 (Saturn)
-       *
-       * @return @b int The appropriate instrument code for the Spacecraft
-       *         Kernel Center ID
-       */
-      virtual int SpkCenterId() const { return 6; }
 
       /**
        *  SPK Reference ID - J2000

